devtools/plugins/desktop/test-env/setup.sh

```diff
@@ -0,0 +1,63 @@
+#!/bin/sh
+
+# Define a function to log steps
+log_step() {
+  echo "\033[1;34m==>\033[0m \033[1m$1\033[0m"
+}
+
+# Define the directories
+ROOT_DIR=$(git rev-parse --show-toplevel)
+BAZEL_OUT_DIR="$ROOT_DIR/bazel-out/darwin-fastbuild/bin"
```

Comment:
```
if [[ $(uname -p) == 'arm' ]]; then
  BAZEL_OUT_DIR="$ROOT_DIR/bazel-out/darwin_arm64-fastbuild/bin"
  else
  BAZEL_OUT_DIR="$ROOT_DIR/bazel-out/darwin-fastbuild/bin"
fi
```
changes for this for m1 macs; otherwise will fail when trying to copy the folder over

Reply:
We can use `BAZEL_OUT_DIR="$ROOT_DIR/bazel-bin"` that should bypass the OS issue (I'll add to the changes)